charts/dsp-argocd/templates/projects/dsp-ci.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
 spec:
   description: CI runners in the broad-dsp-ci cluster; this project managed via dsp-argocd Helm chart
   destinations:
+  - namespace: default
```

Comment:
I'm not grabbing this from `{{ .Values.apps.sysbox.namespace }}` because it is default and that feels dumb, default shouldn't be coupled to sysbox.

Reply:
does sysbox explicitly need to run in default? If so then so be it. If not then slight preference for putting it in it's own namespace.

Reply:
It actually puts everything in kube-system explicitly. I don't think any actual resources will get created in whatever namespace we set here. Should I actually set  kube-system here or try changing their manifests to respect the deployment namespace?

Reply:
In that case you may need to grant the argo project explicit permissions to install to kube-system. I *think* it can only apply resources in namespaces explicitly listed under destinations.

Reply:
Oh then that's a very good point, I'll change the dest. I think their manifests do this as partly a "you can really really only ever have exactly one of these installed at once" deal